libs/wingc/src/type_check.rs

```diff
@@ -2068,8 +2069,9 @@ impl<'a> TypeChecker<'a> {
 				methods,
 				parent,
 				implements,
-				constructor,
+				initializer: constructor,
```

Comment:
style: rename local variable to match (in VS code you can right click on "constructor" in VS Code and click "rename symbol" and type out "initializer" and it should automatically fix it)
```suggestion
				initializer,
```